### PR TITLE
Refresh: Rework First Use tutorial now the IMPORT TWEETS feature is gone

### LINKED ISSUE #29: Drop "import tweets" examples from tutorials
this feature was recently removed from CrateDB. we should no longer mention it in the tutorials

### PR DESCRIPTION
**Pull request by nomicode**
fixes https://github.com/crate/crate-tutorials/issues/29

### CODE REVIEW

File: docs/first-use.rst [hunk 1]
```diff
@@ -38,221 +37,60 @@ You should see something like this:
 
 .. image:: _assets/img/first-use/admin-ui.png
 
+.. SEEALSO
```
Comment (seut) [inline docs/first-use.rst]:
```suggestion
.. SEEALSO::
```

File: docs/first-use.rst [hunk 2]
```diff
@@ -38,221 +37,60 @@ You should see something like this:
...
-
-.. code-block:: psql
-
-   SELECT COUNT(*) FROM tweets
-
-This time, all we want to know is how many tweets we have.
-
-Using `HTTPie`_, you can run this query against CrateDB listening on
-``localhost:4200``, like so:
+   `The CrateDB Admin UI documentation`
```
Comment (seut) [inline docs/first-use.rst]:
```suggestion
   `The CrateDB Admin UI documentation`_
```

File: docs/first-use.rst [hunk 3]
```diff
@@ -38,221 +37,60 @@ You should see something like this:
...
 .. _geospatial: https://crate.io/docs/stable/sql/data_types.html#geo-point
+.. _How-to Guides: https://crate.io/docs/crate/howtos/
 .. _HTTP endpoint: https://crate.io/docs/crate/reference/en/latest/interfaces/http.html
 .. _HTTPie: https://httpie.org/
 .. _install Crash: https://crate.io/docs/clients/crash/en/latest/getting-started.html#install
 .. _object: https://crate.io/docs/crate/reference/en/latest/general/ddl/data-types.html#object
 .. _regular SQL: https://crate.io/docs/crate/reference/en/latest/appendices/sql-compliance.html
 .. _scalar: https://crate.io/docs/stable/sql/scalar.html
 .. _schema: https://crate.io/docs/crate/reference/en/latest/general/ddl/create-table.html#schemas
+.. _The CrateDB Admin UI documentation: https://crate.io/docs/clients/admin-ui/en/latest/
```
Comment (seut) [inline docs/first-use.rst]:
```suggestion
.. _The CrateDB Admin UI documentation: `Admin UI`_
```

**Review by seut — CHANGES_REQUESTED** [review]
Seems like some section/links are broken.

File: docs/first-use.rst (hunk 3 above)
Comment (nomicode) [reply docs/first-use.rst]: woah I had no idea this was possible. I love it! very meta. haha

**Comment by nomicode** [comment]
thx for catching those! committed and rebased

**Review by seut — APPROVED** [review]
👍 thx

**Pull request merged**